technische-universitat-munchen-controlling.csl

```diff
@@ -7,7 +7,7 @@
     <id>http://www.zotero.org/styles/technische-universitat-munchen-controlling</id>
     <link href="http://www.zotero.org/styles/technische-universitat-munchen-controlling" rel="self"/>
     <link href="http://www.zotero.org/styles/universitat-freiburg-geschichte" rel="template"/>
-    <link href="http://www.zotero.org/styles/hwr-berlin" rel="template"/>
+    <link href="http://www.zotero.org/styles/hochschule-fur-wirtschaft-und-recht-berlin" rel="template"/>
```

Comment:
@adam3smith, @zuphilip, only now seeing this, but I'm confused.

This template style was renamed in 2013 (see https://github.com/citation-style-language/styles/commit/d6754738d27fd528775648714ce36ef269373568#diff-2657503fcaf93aaf9836fa01c692099eR22 and https://github.com/citation-style-language/styles/commit/d6754738d27fd528775648714ce36ef269373568#diff-2657503fcaf93aaf9836fa01c692099eR22). Shouldn't our automatic tests have caught this way back? I'm pretty sure we're testing whether the template style exists in the current repository.